js/currencycom.js

```diff
@@ -1165,6 +1165,24 @@ module.exports = class currencycom extends Exchange {
         return this.parseTrades (response, market, since, limit);
     }
 
+    async fetchDepositAddress (code, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const request = {
+            'coin': currency['id'],
+        };
+        const response = await this.privateGetV2DepositAddress (this.extend (request, params));
```

Comment:
Please, add the response sample:

```
~/ccxt $ node examples/js/cli currencycom privateGetV2DepositAddress '{"coin":"USDT"}' --verbose
2022-02-22T04:22:40.548Z
Node.js: v16.13.2
CCXT v1.73.78
currencycom.privateGetV2DepositAddress ([object Object])
fetch Request:
 currencycom GET https://api-adapter.backend.currency.com/api/v2/depositAddress?timestamp=1645503762197&recvWindow=5000&coin=USDT&signature=1d5330ef53548aea15547896b7eed92061e2d049800f9070daedd1895305c228
RequestHeaders:
 { 'X-MBX-APIKEY': '1l98UxKoj85FHu4X' }
RequestBody:
 undefined

handleRestResponse:
 currencycom GET https://api-adapter.backend.currency.com/api/v2/depositAddress?timestamp=1645503762197&recvWindow=5000&coin=USDT&signature=1d5330ef53548aea15547896b7eed92061e2d049800f9070daedd1895305c228 200 OK
ResponseHeaders:
 {
  'Access-Control-Allow-Headers': 'X-MBX-APIKEY, Origin, X-Requested-With, Content-Type, Authorization',
  'Access-Control-Allow-Methods': 'POST, GET, PUT, OPTIONS, DELETE, PATCH',
  'Access-Control-Allow-Origin': '*',
  Connection: 'close',
  'Content-Security-Policy': "connect-src 'self'",
  'Content-Type': 'application/json',
  Date: 'Tue, 22 Feb 2022 04:22:42 GMT',
  Server: 'nginx',
  'Transfer-Encoding': 'chunked',
  Vary: 'Origin, Access-Control-Request-Method, Access-Control-Request-Headers'
}
ResponseBody:
 {"address":"0x97d64eb014ac779194991e7264f01c74c90327f0"}
 ```
 
 ↓
 
 ```JavaScript
        const response = await this.privateGetV2DepositAddress (this.extend (request, params));
        //
        //     { "address":"0x97d64eb014ac779194991e7264f01c74c90327f0" }
        //
 ```
 
Also, let's detach the parsing stage to `parseDepositAddress` (see mentions of `parseDepositAddress` in the other exchanges).

Reply:
oh, that was my mistake, I always include response. missed somehow.. tnx